src/components/victory-area/victory-area.js

```diff
@@ -484,7 +521,8 @@ export default class VictoryArea extends React.Component {
 
     if (animate) {
       const whitelist = [
-        "data", "domain", "height", "padding", "style", "width", "clipWidth", "clipHeight"
+        "data", "domain", "height", "padding", "style", "width",
+        "clipWidth", "clipHeight", "translateX"
```

Comment:
`clipWidth`, `clipHeight` and `translateX` are actually one of it's props. https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-area/victory-area.js#L462-L478 . As the chart initialize https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-area/victory-area.js#L482 `Helpers.modifyProps(assign({clipId}, this.props), fallbackProps, "area");` will assign a default `clipWidth`, `clipHeight` for the chart, and it will createElement using the default props `{React.createElement(this.constructor, props)}`.  

And after the chart is updated (onExit, onEnter, onLoad), `clipWidth`, `clipHeight`, `translateX` is passed as a prop to `VictoryArea`, to achieve the animate effects.